.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,56 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Stellar Docs - Docusaurus/Yarn",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:1-22",
+  "hostRequirements": {
+    "cpus": 2
+  },
+  "forwardPorts": [
+    3000
+  ],
+  "portsAttributes": {
+    "3000": {
+      "elevateIfNeeded": true,
+      "onAutoForward": "openBrowser",
+      "label": "docusaurus",
+      "requireLocalPort": true
+    }
+  },
+  // Configure tool-specific properties.
+  "customizations": {
+    "codespaces": {
+      "openFiles": [
+        "README.md",
+        "devcontainer.md",
+        "docs.log"
+      ]
+    },
+    "vscode": {
+      "settings": {
+        "[typescript]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        },
+        "[json]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        },
+        "[jsonc]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        }
+      },
+      "extensions": [
+        "dbaeumer.vscode-eslint",
+        "orta.vscode-jest",
```

Comment:
i don't think we use jest in this repo. do we need it in the devcontainer?

Reply:
Oh yea nice call out thank you!